faucet/src/http.rs

```diff
@@ -248,9 +257,7 @@ pub async fn start_http(
             let provider = provider.clone();
             let store = store.clone();
             async move {
-                let result = _fund_account_raw(&provider.clone(), params, store, user_allowance, vault_allowance).await;
-                debug!("Result: {:?}", result);
```

Comment:
I think we should keep this, or at least keep a println if it's an error, to be able to print the parse_params error in `_fund_account_raw`